### PR TITLE
scss-lint linting

### PR DESCRIPTION
**Pull request by jackmcpickle**
Update `npm test` with existing scss-linter

Updated the rules too with latest sass guidelines

### CODE REVIEW

**Comment by jackmcpickle** [comment]
This is the simplest way to get linting into the tests. 

TODO - change to either sass-lint or stylelint as scss-lint is ruby based and sass core now moving to Dart. See https://github.com/brigade/scss-lint

I have started setting up stylelint rules, although very configurable, it doesnt seem to cover sass syntax as well as scss-lint and sass-lint.

**Comment by eduardoboucas** [comment]
@jackmcpickle any idea of why the Ruby part is [failing in Travis](https://travis-ci.org/eduardoboucas/include-media/jobs/177373921)? It says it needs Ruby >= 2, even though you're requesting 2.3 in the dependencies.

**Comment by jackmcpickle** [comment]
Not yet. I will keep trying - kind of defeats the point if it doesn't work in travis...

**Comment by jackmcpickle** [comment]
yay - finally travis is green..

after much trial and error - `sudo: required` builds travis outside their container system and with ruby 2...

**Comment by eduardoboucas** [comment]
Thanks! Looks good to me.

@HugoGiraudel 👍 ?

**Comment by jackmcpickle** [comment]
@eduardoboucas you could also activate [HoundCI](https://houndci.com) - uses scss-lint too to automatically comment on PRs and make sure they are following the standard.

**Comment by eduardoboucas** [comment]
Oh, I like the sound of that. Thanks!

**Comment by jackmcpickle** [comment]
While we are on the topic. Another good tool is https://greenkeeper.io/ - I use it on my repos. Creates PRs for you based on packages you have in your package.json file that are out of date and not covered in your semver versioning.

**Comment by eduardoboucas** [comment]
I use Greenkeeper too, but I think it'd be a bit overkill for this project, since we don't have any dependencies other than dev dependencies, which shouldn't be critical anyway.

**Comment by jackmcpickle** [comment]
Fair enough. I'm just a bit of an update junkie. 😸

File: dist/_include-media.scss [hunk 1]
```diff
@@ -237,6 +237,7 @@ $im-no-media-expressions: ('screen', 'portrait', 'landscape') !default;
       $value: get-expression-value($condition, $operator);
 
       // scss-lint:disable SpaceAroundOperator
+      // Long if statement
```
Comment (HugoGiraudel) [inline dist/_include-media.scss]: I’d suggest we just disable that rule entirely. No point in having it if we keep fighting against it.

**Comment by jackmcpickle** [comment]
Updated @HugoGiraudel

**Comment by HugoGiraudel** [comment]
Other than that, it's good to me.

**Comment by eduardoboucas** [comment]
🎉

**Pull request merged**